### PR TITLE
[Unit Tests] Updated all test cases and added test cases for emergency contacts

### PR DESCRIPTION
**Pull request by t-cochran**
**Highlights:**
- All cases that failed were checked and updated
- New test cases were added for: 
    - `Emergency Contact Form` 
    - `Emergency Contact Model`

All tests pass:
image 

**TODO:**
- Integrate our project with travis CI. I set it up on a private repo with this test code merged: 

image

**Note:**
I noticed all tests in `apps/pages/tests/test_views.py` that involve static files (e.g. check if an html template is used) require you to run the command:

`$ python manage.py collectstatic`

- Apparently, the test code checks the `STATIC_ROOT` for static files. This is set in settings.py and behaves as if the app were deployed.

- The collectstatic command creates the `staticfiles/` folder, which we may want to add to .gitignore because it only affects these unit tests and is **not required** to run the app locally, i.e. through `$ python manage runserver`

- More info: [django collectstatic command](https://docs.djangoproject.com/en/3.0/ref/contrib/staticfiles/#management-commands)

**TLDR:**
To run this test code:
`$ python manage.py collectstatic`
`$ python manage.py test`
. . . . . . . . . . . . . test code runs. . . . . . . . .
`$ rm -r staticfiles`

### CODE REVIEW

**Comment by t-cochran** [comment]
**Note:**
I noticed all tests in `apps/pages/tests/test_views.py` that involve static files require you to run the command:

`$ python manage.py collectstatic`

As, apparently, the tests check the `STATIC_ROOT` in settings.py as if the app were deployed. This creates and files the 'staticfiles' folder, which we may want to add to .gitignore. Also, this is something I had to do when I integrated the test code with travis CI.

**TLDR: If you pull this to check the test code:**
`$ python manage.py collectstatic`
`$ python manage.py test`
`. . . . . . . . . . . . . test code runs. . . . . . . . .`
`rm -r staticfiles`

**Comment by t-cochran** [comment]
**Update** After merging #49 I have re-ran these tests and need to go through the ones that have failed:
image
